admin-dev/themes/new-theme/js/pages/product/edit/product-partial-updater.ts

```diff
@@ -267,11 +281,7 @@ export default class ProductPartialUpdater {
     Object.keys(this.initialData).forEach((fieldName) => {
       const fieldValue = this.initialData[fieldName];
 
-      // Field is absent in the new data (it was not in the initial) we force it to empty string (not null
-      // or it will be ignored)
-      if (!Object.prototype.hasOwnProperty.call(currentData, fieldName)) {
-        currentData[fieldName] = '';
-      } else if (_.isEqual(currentData[fieldName], fieldValue)) {
+      if (_.isEqual(currentData[fieldName], fieldValue)) {
```

Comment:
I'm not sure you should change this, there's an explicit comment that explains why this as needed and it's not in the scope of your PR to change this behavior

This extra filling is required when we need to empty/clear some fields